docs/Extensions/ApplicationSearch/README.md

```diff
@@ -28,4 +28,10 @@ Supported operating systems:
 
 -   Windows
 -   macOS
--   Linux (with Cinnamon or Gnome Desktop environment)
+-   Linux (with the following desktop environments)
+    -   GNOME
+    -   KDE
+    -   MATE
+    -   XFCE
+    -   Cinnamon
+    -   Pantheon
```

Comment:
Is this still true? Doesn't it work on all desktop environments now?

Reply:
App icons are only able to be generated on those desktops. Others will have the generic app icon `assets/Extensions/ApplicationSearch/linux-generic-app-icon.png` for every app. It's up to you to decide if that is considered fully functional

Reply:
I would say the extension is fully functional when you can search and start apps. I added a note in the readme though that generic app icons will be used if the app icon can't be extracted.